### PR TITLE
Improved comments in ubiquiti er-x

### LINKED ISSUE #1173: Misleading comment in Ubiquiti ER-X
https://github.com/netbox-community/devicetype-library/blob/bd9fce16f25f0601707dd31faf9430534ddd5da3/device-types/Ubiquiti/ER-X.yaml#L7

should be something like [store.ui.com/collections/operator-edgemax-routers/products/edgerouter-x](https://store.ui.com/collections/operator-edgemax-routers/products/edgerouter-x)

### PR DESCRIPTION
**Pull request by felbinger**
closes #1173

### CODE REVIEW

File: device-types/Ubiquiti/ER-X.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ model: ER-X
 slug: er-x
 u_height: 1
 is_full_depth: false
-comments: '[EdgeRouter X - Advanced Gigabit Ethernet Router](https://www.ui.com/edgemax/edgeswitch-10x/)'
+comments: '[EdgeRouter X - Advanced Gigabit Ethernet Router](https://store.ui.com/collections/operator-edgemax-routers/products/edgerouter-x)'
```
Comment (danner26) [inline device-types/Ubiquiti/ER-X.yaml:7]: I actually think this should be the datasheet: https://dl.ui.com/datasheets/edgemax/EdgeRouter_X_DS.pdf

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
```
weight: 12.01
weight_unit: lb
```

Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

File: device-types/Ubiquiti/ER-X.yaml (hunk 1 above)
Comment (felbinger) [reply device-types/Ubiquiti/ER-X.yaml]: Adjusted

**Comment by felbinger** [comment]
> Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
> 
> ```
> weight: 12.01
> weight_unit: lb
> ```
> 
> Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

Updated, I also added the weight in the given format. I may be a good idea to squash this pr with the title of the first commit.

**Pull request merged**